doc/source/getting_started/launcher.rst

```diff
@@ -0,0 +1,228 @@
+
+
+.. _ref_launch_pymapdl:
+
+
+Launch PyMAPDL
+==============
+
+PyMAPDL can start MAPDL locally, or connect to a session already running locally or in a remote machine.
+
+* `Launch PyMAPDL with a local MAPDL instance`_
+* `Connect PyMAPDL to a local MAPDL instance`_
+* `Connect PyMAPDL to a remote MAPDL instance`_
+
+If you have any problem launching PyMAPDL, see :ref:`Launching issues <ref_launching_issue>`.
+
+
+Launch PyMAPDL with a local MAPDL instance
+------------------------------------------
+
+You can use the :func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>`
+method to have Python start MAPDL and automatically connect to it:
+
+.. code:: pycon
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    >>> mapdl = launch_mapdl()
+    >>> print(mapdl)
+    Product:             Ansys Mechanical Enterprise
+    MAPDL Version:       24.1
+    ansys.mapdl Version: 0.68.0
+
+
+This is the easiest and fastest way to get PyMAPDL up and running. 
+But you need to be able to launch MAPDL locally.
+
+If PyMAPDL cannot find your local installation, please refer to
+`Setting MAPDL location in PyMAPDL`_.
+
+For more information on controlling how MAPDL launches locally, see the
+description of the :func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>` function.
+
+
+Connect PyMAPDL to a local MAPDL instance
+-----------------------------------------
+
+.. _launch_grpc_madpl_session:
+
+Launch a local gRPC MAPDL session
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can start MAPDL from the command line and then connect to it.
+
+To launch MAPDL on Windows (assuming a :file:`C:/Program Files/ANSYS Inc/v241` installation), use:
+
+.. code:: pwsh-session
+
+    C:/Program Files/ANSYS Inc/v241/ansys/bin/winx64/ANSYS211.exe -grpc
+
+To launch MAPDL on Linux (assuming ANSYS is installed in :file:`/usr/ansys_inc` installation), use:
+
+.. code:: console
+
+    /usr/ansys_inc/v241/ansys/bin/ansys211 -grpc
+
+This starts up MAPDL in gRPC mode, and MAPDL should output:
+
+.. code:: output
+
+     Start GRPC Server
+
+     ##############################
+     ### START GRPC SERVER      ###
+     ##############################
+
+     Server Executable   : MapdlGrpc Server
+     Server listening on : 0.0.0.0:50052
+
+You can configure the port that MAPDL starts on with the ``-port`` argument.
+For example, you can start the server to listen for connections at 
+port 50005 with:
+
+.. code:: console
+
+    /usr/ansys_inc/v241/ansys/bin/ansys211 -port 50005 -grpc
+
+
+.. _connect_grpc_madpl_session:
+
+Connect to the local MAPDL instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An MAPDL gRPC server can be connected to from the same host by using:
```

Comment:
```suggestion
An MAPDL gRPC server can be connected to from the same host by using
these commands:
```
In instances like this, I'm never sure whether the sentence should say "... by using these commands" or "... by using this code." Your call!

Reply:
Interesting... Maybe I like more `these commands` but I'm keeping your wording. :)

Reply:
@germa89 @koubaa Here is proof that I question the correct word choice! Because I do strive for consistency, please let me know how you think I should be phrasing this for all PyAnsys libraries!